docs/content/preview/explore/ysql-language-features/pg-extensions.md

```diff
@@ -332,6 +354,35 @@ The `connectby()` function displays a hierarchy of the kind that you see in an _
 
 The `crosstab()`and  `crosstabN()` functions produce "pivot" displays. The _"N"_ in crosstabN() indicates the fact that a few, `crosstab1()`, `crosstab2()`, `crosstab3()`, are provided natively by the extension and that you can follow documented steps to create more.
 
+### postgres_fdw example
+
+First, install the extension:
+
+```sql
+CREATE EXTENSION postgres_fdw;
+```
+
+To connect to a remote YSQL or PostgreSQL database, create a foreign server object. Specify the connection information (except the username and password) using the `OPTIONS` clause:
+
+```sql
```

Comment:
Again, do you need to create the wrapper first?

Reply:
Same as above, `CREATE EXTENSION` creates the fdw under the hood